pkgs/applications/networking/cluster/spacegun/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs, nodejs, stdenv, lib, ... }:
+
+let
+
+  packageName = with lib; concatStrings (map (entry: (concatStrings (mapAttrsToList (key: value: "${key}-${value}") entry))) (importJSON ./package.json));
+
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+in
+nodePackages."${packageName}".override {
+  nativeBuildInputs = [ pkgs.makeWrapper ];
+
+  postInstall = ''
+    # Patch shebangs in node_modules, otherwise the webpack build fails with interpreter problems
+    patchShebangs --build "$out/lib/node_modules/spacegun/node_modules/"
```

Comment:
This fixes errors like this:

```
sh: /nix/store/zay52n90397hcl16g4fp6dahiaiy9p0s-node_mini-media-player-v1.11.0/lib/node_modules/mini-media-player/node_modules/.bin/webpack: /usr/bin/env: bad interpreter: No such file or directory
```

I hope this improves the discoverability (searchengine) of this fix.